File: src/Services/TrainingService/TrainingService.Domain/ClassManager.cs
using System.ComponentModel;
using Education.Core.Domain;
using MongoDB.Bson;

namespace TrainingService.Domain;
[Description("Lớp quản lý")]
public class ClassManager : BaseEntity
{
    public string ClassCode { get; set; }
    public string ClassName { get; set; }
    [Description("Chương trình đào tạo")]
    public string EducationCode { get; set; }

    public ICollection<SemesterClass> SemesterClasses { get; set; } = [];
}

File: src/Webs/daotao/src/domain/class_manager.ts
import {Semesterclass} from "@/domain/semester_class.ts";

export interface ClassManager {
    classCode: string;
    className: string;
    educationCode: string;
    semesterClasses: Semesterclass[];
    id: string;
    createdAt: string; 
    updatedAt: string | null;
}

